eng/versioning/version_client.txt

```diff
@@ -479,6 +476,9 @@ com.azure.resourcemanager:azure-resourcemanager-migration-assessment;1.0.0-beta.
 com.azure.resourcemanager:azure-resourcemanager-databasewatcher;1.0.0-beta.1;1.0.0-beta.2
 com.azure.tools:azure-sdk-archetype;1.0.0;1.2.0-beta.1
 com.azure.tools:azure-sdk-build-tool;1.0.0;1.1.0-beta.1
+com.azure.v2:azure-client-sdk-parent;2.0.0-beta.1;2.0.0-beta.1
```

Comment:
Are we starting these from 2.0.0-beta.1?

Reply:
I think it's the best choice to help differentiate Track 2 and vNext further. Also, I can see a lot of typos happening if both `com.azure:azure-client-sdk-parent` and `com.azure.v2:azure-client-sdk-parent` use `1.x.y`.